src/index.ts

```diff
@@ -31,39 +39,73 @@ export class TestxServer {
   private queries: { [id: string]: any };
   private mutations: { [id: string]: any };
 
+  /**
+   * Create a TestxServer.
+   * @constructor
+   * @param {String} schema - The GraphQL data model
+   */
   constructor(schema: string) {
     this.schema = schema;
   }
 
+  /**
+   * Starts the server.
```

Comment:
I would add a couple of words for the `bootstrap()` method and where the server will be exposed and how to retrieve the server url